### DISCUSSION TITLE
PDF/Doc Upload with OpenAI

### DISCUSSION THREAD

**Discussion by sakib1486** [danny-avila/LibreChat#2108] [Help Wanted]
I was able to attach files through the Assistant API, and I saw a file attachment option available for OpenAI as well which I tried to enable like follows in the librechat.yaml file. But it is still like before where it was able to get images attached but no pdf/doc files which I was looking into. 

**What I Tried**

<img width="711" alt="Screenshot 2024-03-15 130837" src="https://github.com/danny-avila/LibreChat/assets/58849295/62c17e42-10b0-4d88-ba58-7dac93082c0f">

Any idea if this is doable without the assistants at this moment?

**Comment by danny-avila** [comment]
Yes, this is the current functionality. It's coming soon
